wiki/Rules/Content_Voting_Process/id.md

```diff
@@ -0,0 +1,35 @@
+# Proses pengkajian konten
+
+Konten-konten yang diajukan kepada para anggota [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) (BN), [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) (GMT), dan [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) untuk dikaji lebih lanjut akan melalui beberapa tahapan dalam proses pengkajiannya sebagaimana yang tertera di bawah ini.
+
+## Proses kajian
+
+1. Pada mulanya, para mapper, modder, atau pengguna lainnya yang merasa menemukan suatu hal/konten pada beatmap yang disinyalir melanggar [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations) melaporkan hasil temuan mereka melalui situs web Beatmap Nominator.
+2. Laporan yang masuk tersebut akan diperiksa keabsahannya oleh NAT, di mana NAT kemudian akan membuka “berkas kasus” (*content case*) untuk konten yang bersangkutan. Masing-masing anggota BN, GMT, dan NAT kemudian akan menyatakan suaranya secara terpisah terkait apakah konten yang bersangkutan layak guna atau tidak.
+3. Pada saat berkas kasus yang bersangkutan masih terbuka, beatmap-beatmap yang dilaporkan tidak dapat dinominasikan ataupun di-*qualify*. Beatmap-beatmap yang sebelumnya sudah berstatus Qualified akan didiskualifikasi dengan segera.
+4. Berkas kasus yang bersangkutan akan secara otomatis ditutup apabila dalam rentang waktu 3 hari tidak ada suara baru yang masuk atau apabila 7 hari telah berlalu.
```

Comment:
```suggestion
4. Pemungutan suara ditutup apabila dalam rentang waktu 3 hari tidak ada suara baru, atau 7 hari setelah voting dimulai.
```

Reply:
hmm, i don't think i'll switch over to `voting` here as the rest of the article has already been using `pemungutan suara` through and through but i did adapt your suggestion to form a better sentence altogether